src/components/FilterSearch.tsx

```diff
@@ -123,14 +128,33 @@ export function FilterSearch({
   );
 
   useEffect(() => {
+    if (matchingFilters.length > 1 && !onSelect) {
```

Comment:
I feel like it may be beneficial to warn user regardless if onSelect is provided here?

Reply:
this was a deliberate decision because the thinking is that if you provide your own `onSelect` prop, you might be fine with having multiple filters selected in state at once and you wouldn't want this warning to to be logged. it's only assumed to be a problem for the default component behavior. people shouldn't have to fork the component just to remove this warning if they don't want clobbering behavior